dbms/src/Functions/FunctionsString.cpp

```diff
@@ -4392,6 +4392,109 @@ class FunctionRepeat : public IFunction
 };
 
 
+class FunctionSpace : public IFunction
+{
+public:
+    static constexpr auto name = "space";
+
+    static constexpr auto MAX = 16777216;
```

Comment:
Why use this magic number？

Reply:
Because when input bigger than 16777216(32 power of 2) then the space function will return NULL
<img width="659" alt="image" src="https://user-images.githubusercontent.com/3417765/188428689-443137d9-5d7f-4e89-b34c-7a4215af2cec.png">